ServerCore/Helpers/NewFileHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.IO;
+using System.Text;
+
+namespace ServerCore.Helpers
+{
+    public static class NewFileHelper
```

Comment:
How about NewEventFileHelper?

You could also consider making a base class between Create and CreateDemo, which contains this code, and then no helpers would be necessary.

Reply:
Can definitely rename.  Won't do the base class though since we decided that a helper was the way to go when we talked about it due to the demo event wanting these files to be optional.

Reply:
Could still be a method on a base class, always called for non demo events, and optionally called for demo events (though I still wonder if exception handling can replace the use of a demo flag, but let's use the other comment for that discussion)